site/app/libraries/Core.php

```diff
@@ -175,12 +175,11 @@ public function setSessionManager(SessionManager $manager) {
     }
 
     private function createEntityManager(AbstractDatabase $database, ?DebugStack $debug_stack): EntityManager {
-        $config = Setup::createAnnotationMetadataConfiguration(
+        $config = ORMSetup::createAnnotationMetadataConfiguration(
             [FileUtils::joinPaths(__DIR__, '..', 'entities')],
             $this->config->isDebug(),
             null,
-            null,
-            false
+            null
```

Comment:
As we are no now longer overriding the final argument to the `createAnnotationMetadataConfiguration` method, we can get rid of the third and fourth arguments as they default to `null`.

```suggestion
            $this->config->isDebug()
```